app/src/main/java/org/fossasia/phimpme/base/ThemedActivity.java

```diff
@@ -61,8 +61,11 @@ public void updateTheme() {
         coloredNavBar = SP.getBoolean(getString(R.string.preference_colored_nav_bar), true);
         obscuredStatusBar = SP.getBoolean(getString(R.string.preference_translucent_status_bar), true);
         applyThemeImgAct = SP.getBoolean(getString(R.string.preference_apply_theme_pager), true);
-        setNavigationBarColor(getPrimaryColor());
-        setNavBarColor();
+        if(coloredNavBar)
+        {
```

Comment:
I've it inline

Reply:
how can you  inline it without else statement?

Reply:
Ok, the curly bracket and the if statement should be on the same line, I thought you were talking about the concept of inline functions and that got me confused.